romancal/regtest/test_tweakreg.py

```diff
@@ -81,6 +81,48 @@ def test_tweakreg(
 
     assert rms < 1.3 / np.sqrt(2)
 
+    # check if the Mean Absolute Error is less that 10 milliarcsec  (DMS406)
+    abs_diff = (np.absolute(diff)) / 1.0e-3
+    mean_abs_error = (tweakreg_out.meta.wcs_fit_results.mae) / 1.0e-3
+    assert mean_abs_error < 10.0
+    assert np.max(abs_diff) < 10.0
+    passmsg = "PASS" if np.max(abs_diff) < 10.0 else "FAIL"
+    dms_logger.info(
+        f"DMS406 {passmsg} the Absolute Astrometric Uncertainty of {np.max(abs_diff):5.2f} mas is less that 10 mas."
+    )
+
+    # check that tweakreg has been run
+    if tweakreg_out.meta.cal_step.tweakreg == "COMPLETE":
+        # Find the reference catalog used by tweakreg
+        for entry in tweakreg_out.meta.cal_logs:
+            refcat_name = " "
+            if "abs_refcat" in entry:
+                log_substring = entry[entry.rfind("abs_refcat") :]
+                refcat_name = log_substring[: log_substring.find("\n")]
+
+        assert "GAIA" in refcat_name
+        passmsg = "PASS" if "GAIA" in refcat_name else "FAIL"
+        dms_logger.info(
+            f"DMS549 MSG: {passmsg}, {refcat_name} used to align data to "
+            f"the Gaia astrometric reference frame."
+        )
```

Comment:
```suggestion
    assert tweakreg_out.meta.cal_step.tweakreg == "COMPLETE"
    # Find the reference catalog used by tweakreg
    for entry in tweakreg_out.meta.cal_logs:
        refcat_name = " "
        if "abs_refcat" in entry:
            log_substring = entry[entry.rfind("abs_refcat") :]
            refcat_name = log_substring[: log_substring.find("\n")]

    assert "GAIA" in refcat_name
    passmsg = "PASS" if "GAIA" in refcat_name else "FAIL"
    dms_logger.info(
        f"DMS549 MSG: {passmsg}, {refcat_name} used to align data to "
        "the Gaia astrometric reference frame."
    )
```
Having this inside of an `if` means that if tweakreg doesn't "COMPLETE" the assert will never be tested and the test can pass even if it doesn't meet the requirement.